engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -608,7 +612,7 @@ case class WorkflowExecutionActor(params: WorkflowExecutionActorParams)
 
       // Any other case is unexpected and is considered a failure (we should not be trying to start a job in any other state)
```

Comment:
the updated wording makes it sound like this is not so unexpected anymore so perhaps the comment should be adjusted as well